src/app/components/form/form.component.scss

```
@@ -0,0 +1,34 @@
+@import "src/styles/styles.scss";
+
+.content {
+  max-width: 2000px;
+  width: 100%;
+}
+
+.form-content {
+  background-color: $color-grey;
+  border-top: 8px solid $color-brand-blue;
+  margin-top: 50px;
+  max-width: 100%;
+  width: $width-form;
+  
+  .form {
+    margin: 24px;
+    max-width: 252px;
```

```suggestion
    max-width: 250px;
```

Remember to use a base value, can be 5px